### ISSUE TITLE
Query scrollbars are VERY hard to use

### ISSUE THREAD

**Issue by brobichaud** [Microsoft/azuredatastudio#5139]
Issue Type: <b>Bug</b>

When performing a query, especially one with multiple output result-sets the scrollbars are really hard to find and use. They are too small and subtle, and often don't appear when I expect them too. I've struggled with scrolling in this UI far more than any other app I've used in the past decade or more.

Azure Data Studio version: azuredatastudio 1.6.0 (eb35dae1d1238e03f1cab3bd45e151f381b22aef, 2019-04-17T07:11:15.944Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7600U CPU @ 2.80GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.88GB (6.59GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details>Extensions: none

**Comment by DTronD** [comment]
Amen to @brobichaud  for raising this.  It seems like such a small thing, but it's like a pebble in your shoe:  the longer it goes, the more it bugs the hell outta ya.

So it seems that the setting for controlling the horizontal scroll bar height is ignored in ADS.  I guess it's a carry-over from the underlying VSCode base, but at least there is an existing setting that could be used for this enhancement. Please.

"editor.scrollbar.horizontalScrollbarSize": 20,
"editor.scrollbar.horizontal": "visible"

**Comment by claytonjhamilton** [comment]
Agree. Very annoying. Especially when the wider scrollbar version is just staring at me in the VS Code IDE.

**Comment by beautifulcoder** [comment]
I have this set in Azure Data Studio but looks like the grid simply ignores these settings. Is the fix to allow customizations of the scrollbar width? I assume for both horizontal and vertical scrollbars.

"editor.scrollbar.verticalScrollbarSize": 6

**Comment by beautifulcoder** [comment]
Oof, I am noticing it also ignores this setting, and the scrolling experience is clunky.

"editor.mouseWheelScrollSensitivity": 3

**Comment by mejobloggs-cw** [comment]
Any progress with this? The results/query horizontal scrollbar is still a terrible experience. I can't find a way to change their color.

Currently multiple times a day I find myself hunting for them
